kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -897,6 +897,55 @@ object KyuubiConf {
       .stringConf
       .createOptional
 
+  val ENGINE_TRINO_CONNECTION_PASSWORD: OptionalConfigEntry[String] =
+    buildConf("kyuubi.session.engine.trino.connection.password")
+      .doc("The password used for connecting to trino cluster")
```

Comment:
fill password in URL but not `beeline -p` is not intuitive i think

Reply:
> fill password in URL but not `beeline -p` is not intuitive i think

Yes, but Kyuubi server authentication uses `-P`. And the passwords of server and engine are different at most time.